atak-civ/plugin-examples/plugintemplate/app/src/main/java/com/atakmap/android/plugintemplate/ActiveTimer.java

```diff
@@ -24,19 +24,23 @@
     public enum ActiveTimerState { RUNNING, PAUSED, FINISHED, DISMISSED}
 
     private Timer timer;
-    private int remainingDurationMillis;
+    private long remainingDurationMillis;
     private CountDownTimer countDown;
     private ActiveTimerState state;
+    public TimerListAdapter containingAdapter;
+    public Context context;
```

Comment:
I think we should keep them public because I may need to reference the context externally to play the sound and reference the containingAdapter if I need to update it from another file. I will change these if we discover that they don't end up needing to be public later on though.

As for the context in makeSound I'll fix that now